apps/cms-v2/src/modules/providers/contentful/contentful-client.ts

```diff
@@ -232,6 +276,7 @@ export class ContentfulClient {
         return this.uploadProductVariant({ configuration, variant });
       }
     } catch (err) {
+      logger.error("Error during the upsert", { error: err });
```

Comment:
question: Errors during call to cms provider should be log as error?

Reply:
not exactly. Now you can log it as error, but in next PRs you should introduce proper error handling.

If contentful returns something like 401/403 (credentials) - its rather warning, because user hasn't configured app proeprly

If contentful returns 500 - they are down. We can log this as warning, but we can't do anything about it (but good to know it happened)

If contentful returns something like bad request → likely means app has invalid format of data. This is app fault, it should be error/sentry etc